protocol/0009-MRKP-mark_price.md

```diff
@@ -6,70 +6,81 @@ A *Mark Price* is a concept derived from traditional markets.  It is a calculate
 
 Introduce a network parameter `network.markPriceUpdateMaximumFrequency` with minimum allowable value of `0s` maximum allowable value of `1h` and a default of `5s`.
 
-## Acceptance criteria
-
-- The mark price must be set when the market leaves opening auction. (<a name="0009-MRKP-002" href="#0009-MRKP-002">0009-MRKP-002</a>)
-- Each time the mark price changes the market data event containing the new mark price should be emitted. Specifically, the mark price set after leaving each auction, every interim mark price as well as the mark price based on last trade used at market termination and the one based on oracle data used for final settlement should all be observable from market data events. (<a name="0009-MRKP-009" href="#0009-MRKP-009">0009-MRKP-009</a>)
+The *Mark Price* represents the "current" market value for an instrument that is being traded on a market on Vega. It is a calculated value primarily used to value trader's open portfolios against the prices they executed their trades at. Specifically, it is used to calculate the cash flows for [mark-to-market settlement](./0003-MTMK-mark_to_market_settlement.md).
 
-Algorithm 1:
+The mark price is instantiated when a market opens via the [opening auction](./0026-AUCT-auctions.md).
 
-- If `network.markPriceUpdateMaximumFrequency=0s` then any transaction that results in one or more trades causes the mark price to change to the value of the last trade and only the last trade. (<a name="0009-MRKP-003" href="#0009-MRKP-003">0009-MRKP-003</a>)
```

Comment:
I think this is no longer allowed (we don't use it and it would cause division by `0` in the new methodology).

Reply:
So that AC is gone I believe (cc @ze97286 ).